pkg/kubernetes/scripts/virtual-machines/index.jsx

```diff
@@ -19,25 +19,22 @@
 
 import 'regenerator-runtime/runtime' // required for library initialization
 import React from 'react'
-import { createStore, applyMiddleware, compose } from 'redux'
-import createSagaMiddleware from 'redux-saga'
+import { createStore } from 'redux'
 import { Provider } from 'react-redux'
 
+import { initMiddleware } from './kube-middleware.jsx'
 import reducers from './reducers.jsx'
-import rootSaga from './sagas.jsx'
 import * as actionCreators from './action-creators.jsx'
 import VmsListing from './components/VmsListing.jsx'
 
-const sagaMiddleware = createSagaMiddleware()
```

Comment:
Heh, so no empty sagas after all :-)

Reply:
Removed within a separate commit so they might enjoy comeback easily. Anyway the use await/async seems to be better and fully sufficient.